Makefile

```diff
@@ -85,13 +99,23 @@ build: # Build REANA client and cluster components.
 	reana-dev install-client && \
 	reana-dev install-cluster && \
 	reana-dev git-submodule --update && \
-	reana-dev docker-build -t latest && \
-	reana-dev git-submodule --delete
+	reana-dev docker-build -b DEBUG=${DEBUG} && \
+	reana-dev git-submodule --delete && \
+	if [ "${DEBUG}" -gt 0 ]; then \
+		echo ${ccyellow} && \
+		echo "Please run minikube mount in a new terminal to have live code updates." && \
+		echo "minikube mount $$(pwd)/..:/code" && \
```

Comment:
Again here put `$ ... ` to indicate console session and make sure it is separated by empty lines to clearly delineate it from the explanatory text.